### ISSUE TITLE
Homebrew Bazel doesn't run tools/bazel

### ISSUE THREAD

**Issue by dfabulich** [bazelbuild/bazel#1503]
The Bazel install scripts and `.deb` files install a wrapper script that searches for `tools/bazel`, falling back to running `../lib/bazel-real`.

The Bazel homebrew script doesn't work that way; homebrew's `bazel` is the binary executable.

This is because the Bazel homebrew script runs `./compile.sh`, the output of which is the Bazel binary, not the Bazel script.

**Comment by dfabulich** [comment]
Filed https://github.com/Homebrew/homebrew-core/pull/2921 to address this

**Comment by dfabulich** [comment]
@damienmg What's the status of this bug? I made [a PR to fix it for 0.3.0](https://github.com/Homebrew/homebrew-core/pull/2921), but when you said [you'd make your own PR](https://github.com/Homebrew/homebrew-core/pull/1887#issuecomment-231984036), my PR was closed in favor of your forthcoming PR.

Am I right in thinking that your PR will require somebody to do a Bazel release, in order to release the fixes for issue #1504? Are you waiting to deliver Bazel 0.3.1 before filing the PR?
